src/sections/forms/styles.css

```diff
@@ -0,0 +1,87 @@
+.react-rainbow-admin-forms_container {
+    min-height: 100vh;
```

Comment:
update master because we change layout styles and now this is not needed, review this